datafusion/expr/src/logical_plan/extension.rs

```diff
@@ -164,6 +164,26 @@ pub trait UserDefinedLogicalNode: fmt::Debug + Send + Sync {
     /// Note: [`UserDefinedLogicalNode`] is not constrained by [`Eq`]
     /// directly because it must remain object safe.
     fn dyn_eq(&self, other: &dyn UserDefinedLogicalNode) -> bool;
+
+    /// Serialize this node to a byte array. This serialization needs not to include
```

Comment:
I think it might be clearer to say that the serialization *should not* include input plans (as they are serialized by the overall plan serializer, right?)